src/v3/src/contexts.ts

```diff
@@ -22,7 +22,7 @@ import {
 type IWidgetContext = {
   authClient: OktaAuth;
   widgetProps: WidgetProps;
-  setMessages: StateUpdater<IdxMessage[]>;
+  setMessage: StateUpdater<IdxMessage | undefined>;
```

Comment:
I think there's a utility type for this

Reply:
Yea, Lester told me not to use that utility anymore :( so I have been slowly removing and replacing them, and avoid using them until we can completely remove them.